content/ember-4-0-released.md

```diff
@@ -0,0 +1,244 @@
+---
+title: Ember 4.0 released
+authors:
+  - jen-weber
+  - matthew-beale
+date: 2021-11-22T00:00:00.000Z
+tags:
+  - releases
+  - '2021'
+  - version-4-x
+---
+
+Today the Ember project is releasing version 4.0 of Ember.js, Ember Data, and
+Ember CLI. This major version bump focuses
+the framework by removing long-deprecated APIs and support for legacy platforms.
+
+Although [Ember "Octane"](https://emberjs.com/editions/octane/) APIs have been
+the default for new applications since Ember 3.15, the framework has continued
+to support "Classic" framework features in accordance with our semantic
+versioning commitment. Ember 4.0 takes a step forward and drops
+already-deprecated classic APIs, however the foundational `EmberComponent` and
+`EmberObject`/`computed` APIs are not removed in this release.
+
+This blog post will help you understand how to upgrade and navigate any breaking
+changes.
+
+Additionally, today we're promoting Ember 3.28 to become Ember's latest
+Long-Term Support (LTS) release. Ember's LTS release process provides bugfix
+support for about 36 weeks, and security patches for about 54 weeks. See the
+[3.28 release post](https://blog.emberjs.com/ember-3-28-released) and [LTS
+documentation](https://emberjs.com/releases/lts/) for more details.
+
+For applications using LTS releases, upgrading directly to 4.0 is not
+encouraged. Best practice for those applications is to upgrade to Ember 3.28 LTS
+as soon as possible, then wait for the first LTS of the 4.0 series to be
+released. Ember 4.4 will be the first candidate.
+
+## Major Releases in Ember
+
+Why are there no new features in this release?
+This has been the tradition for all major Ember releases since v2.
```

Comment:
A nitpick. The word `all` may sound weird because there have been only 2 major versions (not several). I think it's okay to leave the word out.

```suggestion
This has been a tradition for major Ember releases since v2.
```